Tests/conf.json

```
@@ -20,6 +20,10 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": "Twitterv2",
+            "playbookID": "playbook-Test_Twitter_v2"
```

The playbook id field in the playbook and the playbook id here are not the same. please change. 

```suggestion
            "playbookID": "playbook-Test_Twitter v2"
```